### PR TITLE
Services

### PR DESCRIPTION
**Pull request by wiliajc87**

### CODE REVIEW

**Comment by SStrausman** [comment]
Looks good to me.

File: app/controllers/services_controller.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    @service = Service.new
+  end
+
+  def create
+    @service = Service.new(:service_params)
+    @service.save
+  end
+
+  def update
+    @service = Service.find(params[:id])
```
Comment (schmidtsusanr) [inline app/controllers/services_controller.rb:16]: @wiliajc87  With the update method, doesn't there need to be a line updating the params?

**Comment by tjmasterson** [comment]
looks good man!

File: app/controllers/services_controller.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  def index
+    @services = Service.all
+  end
+
+  def new
+    @service = Service.new
+  end
+
+  def create
+    @service = Service.new(:service_params)
```
Comment (schmidtsusanr) [inline app/controllers/services_controller.rb]: @wiliajc87 Should service_params (line 11) be a symbol?

**Pull request merged**